frontend/src/components/AnnouncementsList/AnnouncementsList.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import Announcement from '../Announcement/Announcement';
+import PropTypes from 'prop-types';
+
+const AnnouncementsList = ({ boardsIds, announcementsNumber, showBoardNames }) => {
+  const { data, isLoading, isError } = useQuery('boardsList', async () => {
+    return await fetch('/boards').then((response) => response.json());
+  });
+
+  const [announcements, setAnnouncemensts] = useState([]);
+  const [windowSize, setWindowSize] = useState('');
+
+  useEffect(() => {
+    const resizeHandler = () => setWindowSize(window.innerWidth);
+    window.addEventListener('resize', resizeHandler);
+
+    return () => {
+      window.removeEventListener(resizeHandler);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (!!data && !!data.boards) {
+      console.log(data.boards.filter(({ boardId }) => boardsIds.includes(boardId)));
+      const announcementsToShow = data.boards
+        .filter(({ boardId }) => boardsIds.includes(boardId))
+        .reduce((acc, { announcements }) => {
```

Comment:
Wiem, że `acc`, `i`, i inne są stosowane, ale raczej powinniśmy używać pełnych nazw. Są czytelne wtedy.
Zamiast acc daj `accumulator`, ale czy na pewno? Czym jest `accumulator`?
albo lepiej daj nazwę taką, czym to jest, bo ja po nazwie nie wiem, czym jest akumulator.
Wiem, ze po kodzie mogę poznać, ale łatwiej jest, jak nazwa przedstawia, co to jest, np: `allAnnouncements`.?

Reply:
done